.travis.yml

```diff
@@ -22,4 +22,4 @@ script:
      export PATH=$(npm bin):$PATH &&
      python3 setup.py install &&
      cd ../coala &&
-     coala --ci"
+     coala --ci';
```

Comment:
Does these commits needs to here, although i am totally fine with these changes but I am confused whether it should be here or be in a new issue

Reply:
@RaiVaibhav This issue is about adding coala file, but as @jayvdb mentioned, I need to make it work on CI.
Since Travis isn't showing logs of coala. I need help to print it.

Reply:
@Makman2 @jayvdb  need to know does it need to create `CI Test` regarding `.coafile` like [same it done on coala/coala](https://github.com/coala/coala/blob/9de6241bc8cebcf32bffe773c6ebbb5018bfad6b/tests/coalaCITest.py)

Reply:
The Travis file is all linked with .coafile, it is just not printing output that is given by anything inside
```
docker run rultor-python sh -c " "
```
Also, @jayvdb shouldn't we fix Travis link in another PR?

Reply:
Previously I was saying don't we need to check wherether `.coafile` exists or not???

Reply:
IMO It should have to check

Reply:
@RaiVaibhav Why do we need to check for the existence of  `.coafile`? 😕

Reply:
It was my opinion that if we are creating a `.coafile` then we also have to check `.coafile` exist or not 
Its like a enhancement, It will be good if `Travis CI` also keep a track record of `.coafile` existence of a  `REPO` , its not like a necessity my review was my first comment.